libs/core/src/lib/status-indicator/status-indicator.component.scss

```diff
@@ -2083,27 +2078,30 @@
   box-sizing: inherit;
   font-size: inherit
 }
+.fd-status-indicator--link {
+  cursor: pointer
+}
 .fd-status-indicator--positive {
   fill: #107e3e;
   fill: var(--sapPositiveElementColor,#107e3e)
 }
-.fd-status-indicator--positive__text {
+.fd-status-indicator__text--positive {
   color: #107e3e;
   color: var(--sapPositiveElementColor,#107e3e)
 }
 .fd-status-indicator--critical {
   fill: #e9730c;
   fill: var(--sapCriticalElementColor,#e9730c)
 }
-.fd-status-indicator--critical__text {
+.fd-status-indicator__text--critical {
   color: #e9730c;
   color: var(--sapCriticalElementColor,#e9730c)
 }
 .fd-status-indicator--negative {
   fill: #b00;
   fill: var(--sapNegativeElementColor,#b00)
 }
-.fd-status-indicator--negative__text {
+.fd-status-indicator__text--negative {
```

Comment:
This is again the same discussion we have in fund-styles PR. 
You have the following markup:
```
<div class="fd-status-indicator fd-status-indicator--negative">
    <span class="fd-status-indicator__text--lg fd-status-indicator__text--negative">35% </span>
</div>
```
You have the `--negative` modifier applied on two places which is not ok. 
You need to set it only on the parent level and then use specificity to change the color. It's not the same as for the size, as size can be different from the size of the parent (which I doubt will happen) but the color of the label should be the same as the color of the status indicator.

So you need to have:
```
.fd-status-indicator--negative {
    .fd-status-indicator__text {
         color: #b00;
         color: var(--sapNegativeElementColor,#b00)
     }
}

.fd-status-indicator--positive {
    .fd-status-indicator__text {
         color: #107e3e;
         color: var(--sapPositiveElementColor,#107e3e)
     }
}
```

This way you will have a modifier only on the parent level:
```
<div class="fd-status-indicator fd-status-indicator--negative">
    <span class="fd-status-indicator__text--lg">35% </span>
</div>
```